_health-care/_js/_form.js

```diff
@@ -1241,6 +1285,9 @@ var HealthApp = (function() {
           e.preventDefault();
           // TODO(awong): Remove this debugging logic when things work.
           console.log(build1010ezXml(formRoot));
+          for (var i = 0; i < allInputs.length; i++) {
+            validate(allInputs[i]);
```

Comment:
This doesn't actually control what happens to the data if the validations pass or fail yet. I'm guessing instead of running validations on submit of the whole form we'll instead want to run them on "Continue" to the next set of questions.